tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/dimension/element-exists-with-all-attributes.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity> <!-- at least 1 -->
+    <iati-identifier></iati-identifier>  <!-- 1 and only 1-->
```

Comment:
According to the new docs format, we need to remove all these `<!-- 1 and only 1-->` comments